airqo-platform/AirQo-frontend
# [NetManager] Set location defaults on chart forms

#### Summary of Changes (What does this PR do?)
- Move locationFilterOptions to store
- Set defaults on chart forms

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


Good to have (up to reviewer's discretion):

#### How should this be manually tested?
* Pull and checkout to this [branch](ft-set-location-defaults-plat-61) locally
* Install node requirements `npm install`
* Start the application `npm run prod-mac` (on mac) or `npm run prod-pc` (on windows platform)
* On the dashboard, ensure the locations dropdown shows suggestions
* On clicking `Customise chart` on the chart dropdown, all the form fields must be pre-filled.
#### What are the relevant tickets?
- [PLAT-61](https://airqoteam.atlassian.net/browse/PLAT-61)

#### Screenshots (optional)
